Microsoft.Toolkit.Uwp.UI.Controls.Design/BladeItemMetadata.cs

```diff
@@ -32,9 +32,6 @@ public BladeItemMetadata()
                     b.AddCustomAttributes(nameof(BladeItem.TitleBarVisibility),
                         new CategoryAttribute(Properties.Resources.CategoryCommon)
                         );
-                    b.AddCustomAttributes(nameof(BladeItem.Title),
```

Comment:
Does Header not need to take its place here?

Or is that taken care of by the Expander inheritance in this situation?

Reply:
That should be part of the Expander